vignettes/shinyMobile.Rmd

```diff
@@ -326,7 +326,8 @@ card(
     "app",
     header = FALSE
   ),
-  full_screen = TRUE
+  full_screen = TRUE,
+  style = "width: 322px;"
```

Comment:
makes it more mobile-like 😎 iPhone 15 has 393px × 852px viewport, so I scaled it so we have 322px x 700px. If we would set it to exactly to the viewport there will be annoying scrolling issues on mobile.

Reply:
Thanks for this change. Could we center the iframe container?

Reply:
Looking back at it, the width seems too narrow and this does not feel natural. Maybe we should use the correct height instead.